docs/ruleset_functions/get_HVAC_building_area_types_and_zones.md

```diff
@@ -0,0 +1,196 @@
+# get_HVAC_building_area_types_and_zones
+**Scheme Version:** 0.0.23  
+
+**Description:** Get the hvac_ids associated with each building area type associated with U_RMR, P_RMR, or B_RMR. Also returns the total floor area of each building area type
+- used to verify the correct type of HVAC baseline system (or systems)
+- HVAC building area type is determined first by looking at the building segment HVAC_BAT.  If this variable does not exist, the function next looks at building segment lighting_building_area_type.  If this variable doesn't exist, the function will look at the space lighting type.  Support spaces such as corridors, stairwells and other support spaces will inherit the predominant building area type EXCEPT in buildings where the predominant HVAC_BAT is residential.  In residential buildings, support spaces such as corridors and stairwells will be classified as Other Non-Residential.
+- the entry "CLASSIFICATION_SOURCE" in the **building_area_types_with_total_area_and_zones_dict** passes information about how the HVAC_BAT was determined for each segment along

```

Comment:
In line 6 we say 'In residential buildings, support spaces such as corridors and stairwells will be classified as Other Non-Residential.'.
90.1-2022 has clarified that such support spaces are called 'residential associated HVAC zone' and classified as residential. However, i am not sure if 90.1 2019  is clear about how such spaces shall be classified. am i missing some requirmeent?

Reply:
the way the function is written, if a building segment has the HVAC_BAT "RESIDENTIAL" - the spaces will be classified as residential, regardless of the space type.  However, if there is no user-given HVAC_BAT, these spaces will be classified as OTHER_NON_RESIDENTIAL - I've edited the description for clarity.  Is this consistent with the requirements of ASHRAE 90.1?

Reply:
I am not sure. Might need to be clarified by @mkarpman or @JasonGlazer

Reply:
Corridors and other non-residential spaces in residential buildings are considered non-residential. This was discussed in comments to line 194. There was an official interpretation for this, and it is also (somewhat) consistent with 90.1 2022 where such spaces are also not modeled with System 1/2. I beleive this is how it's currently done in the RDS (line 182)